akka-remote/src/main/protobuf/WireFormats.proto

```diff
@@ -64,7 +64,15 @@ message PropsData {
   required DeployData deploy = 2;
   required string clazz = 3;
   repeated bytes args = 4;
-  repeated string classes = 5;
+  // serialized props parameters
+  // older wire protocol: contains class name for each arg
+  // newer wire protocol: contains string manifest for each arg
+  repeated string manifests = 5;
+  // newer wire protocol: serializer id for each arg
+  repeated int32 serializerIds = 6;
```

Comment:
can these two extracted into a record? That will enforce equal lengths of these sections (we can't do that for the manifests field ofc)